datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/config/AppConfigResolver.java

```diff
@@ -193,6 +209,20 @@ public CompletableFuture<AppConfig> get(final DataFetchingEnvironment environmen
     }
     appConfig.setSearchBarConfig(searchBarConfig);
 
+    final HomePageConfig homePageConfig = new HomePageConfig();
+    try {
+      homePageConfig.setFirstInPersonalSidebar(
+          PersonalSidebarSection.valueOf(_homePageConfig.getFirstInPersonalSidebar()));
```

Comment:
interesting. we had to configure just the first thing in the sidebar?

Reply:
yeah the request was to change what the first section in the sidebar on the home page - specifically the request is that they want to have the "my groups" tab first, but wanted to make it configurable so we don't have to do this for everyone if we don't want to